Makefile

```diff
@@ -133,7 +137,7 @@ docs:
 ##@ Testing:
 
 .PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
+test: manifests generate generate-mock fmt vet envtest ## Run tests.
```

Comment:
should we add generate-mock as a sub target of `generate` so whether we call build or test these files get generated?

Reply:
Yeah if we make a new `generate` target and rename the existing one